### DISCUSSION TITLE
Is s curve acceleration already enabled? im not sure if i should be touching config.h or config.adv.h

### DISCUSSION THREAD

**Discussion by Tapap84** [Jyers/Marlin#1660] [General Discussion]
I seen a post the other day talking about s curve acceleration and it seemed interesting. I just got to the point of compiling my own firmware but I'm still a ways from understanding it all. I don't know if I should be messing in config.h or config.adv.h.

v2.0.1 with manual 3x3 template. 
also still unsure if i should use that or use the jyers template and just set manual mesh up

in config.adv.h on line 2068 there's //#define BEZIER_CURVE_SUPPORT. does that need any additional info with it or just uncheck it?

in config.h line 1017 looks enabled with #define S_CURVE_ACCELERATION

My goal is to try to smooth out some motions on my printer. For models I run stock 500 acceleration and 8 jerk and for parts that need to be as accurate as possible I use 1000 acceleration and 15 jerk. My 500/8 will produce buldged corners while 1000/15 will be sharp but ringing just does start. I don't know if these guys are fabrication the truth a little but I'm seeing post from people with extremely sharp calibration cubes with absolutely no ringing. They claim they're not running linear advance, they're using marlin, it's a stock machine, and they're running values like 1000/15.

**Comment by Cal-LF** [comment]
Jyers template is for info only. You will likely not have his setup. S curve is enabled, BEZIER_CURVE_SUPPORT is not but could be tried. Not sure it is needed though.
